dataprofiler/tests/profilers/profiler_options/test_profiler_options.py

```diff
@@ -150,7 +158,8 @@ def test_disabling_all_stats(self, *mocks):
         float_options = options.structured_options.float.properties
         int_options = options.structured_options.int.properties
         for option in ["histogram_and_quantiles", "min", "max", "sum",
-                          "variance"]:
+                          "variance","skewness", "kurtosis",
+                       "num_zeros", "num_negatives"]:
```

Comment:
Looks like the formatting got a bit messed up here, should align the start of each line with the `"` in front of `histogram_and_quantiles`